### DISCUSSION TITLE
Add tag aliases

### DISCUSSION THREAD

**Discussion by qingshuizheng** [ahyatt/ekg#23] [Ideas]
Hi @ahyatt , nice to have this new discussion section! 

Are we gonna have the tag aliases like `org-roam-alias`. It seems reasonable to have aliases for this use case or the like: "San Zhang", or "Zhang San", both point to the same person whose name is "张三" in Chinese. 

Thanks,
Zheng

**Comment by ahyatt** [comment]
Seems like a reasonable idea, thanks for bringing it up.  I think we'd have to do this by having schema for tags that are aliases of other tags.  Then when you retrieve the notes for a tag, you retrieve all the aliases. Or, a tag could have aliases just for completion, but when you select, it's always the canonical tag name.   I think that's an easier thing to implement, and less likely to lead to problems down the road.

**Reply by qingshuizheng** [reply]
Nice to hear that. 
Take your time and do it in an approach for the best of future-proof development of ekg. 😄 

Thanks for the great package.
